docs-devsite/ai.prebuiltvoiceconfig.md

```diff
@@ -0,0 +1,43 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# PrebuiltVoiceConfig interface
+> This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
+> 
+
+Configuration for a pre-built voice.
+
+<b>Signature:</b>
+
+```typescript
+export interface PrebuiltVoiceConfig 
+```
+
+## Properties
+
+|  Property | Type | Description |
+|  --- | --- | --- |
+|  [voiceConfig](./ai.prebuiltvoiceconfig.md#prebuiltvoiceconfigvoiceconfig) | string | <b><i>(Public Preview)</i></b> The voice name to use for speech synthesis.<!-- -->See https://cloud.google.com/text-to-speech/docs/chirp3-hd for names and sound demos. |
```

Comment:
For a full list of names and demos of what each voice sounds like, see [Chirp 3: HD voices](https://cloud.google.com/text-to-speech/docs/chirp3-hd).